dandi/cli/tests/test_register.py

```diff
@@ -0,0 +1,78 @@
+import re
+from os.path import dirname
+from pathlib import Path
+from traceback import format_exception
+
+from click.testing import CliRunner
+import pytest
+import yaml
+
+from ..command import register
+from ...consts import dandiset_identifier_regex, dandiset_metadata_file
+
+
+def yaml_load(s):
+    obj = yaml.load(s, Loader=yaml.BaseLoader)
```

Comment:
note: in some other code places we use `yaml.safe_load` which does not demand `Loader`, but I guess it would be suboptimal here (and thus avoiding need for `yaml_load`) since we would not test an entire document... something to keep in mind I guess (unless may be BaseLoader also loads just first document? I didn't check)